keyboards/handwired/dactyl_coolio/keymaps/id-b3/rules.mk

```diff
@@ -0,0 +1,25 @@
+OLED_DRIVER_ENABLE = yes   # Enables the use of OLED displays
+ENCODER_ENABLE = yes       # Enables the use of one or more encoders
+RGBLIGHT_ENABLE = yes      # Enable keyboard RGB underglow
+MOUSEKEY_ENABLE = no  		# Mouse keys(+4700), needed for baller
+POINTING_DEVICE_ENABLE = no
+EXTRAKEY_ENABLE = yes
+
+
+TAP_DANCE_ENABLE = yes     # Enable tap dancing
+
+LTO_ENABLE = no
+UNICODE_ENABLE   = no  # Unicode
+SWAP_HANDS_ENABLE= no  # Allow swapping hands of keyboard
+NO_USB_STARTUP_CHECK = yes
+STENO_ENABLE = no
+BOOTMAGIC_ENABLE =no
+TERMINAL_ENABLE = no
+GRAVE_ESC_ENABLE = no
+MAGIC_ENABLE = no
+SPACE_CADET_ENABLE = no
+KEY_LOCK_ENABLE = no
+AUDIO_ENABLE = no
+CONSOLE_ENABLE = no
+VELOCIKEY_ENABLE = no
+COMMAND_ENABLE   = no # Commands for debug and configuration
```

Comment:
You probably don't need a lot of this; most features are disabled by default.

Reply:
Yes you're correct. I've included all these explicitly as I was searching for 20-30 bytes of space to fit all the features I wanted on a pro-micro (i.e. an animated OLED screen). I'll see which ones actually impacted the decrease in firmware size and remove the rest.